scripts/create_aggregated_zone_config.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+"""
+Create an aggregation zone config file by looking up sub zones based on the zone name.
+Sums capacities, all contributors.
+poetry run python scripts/create_aggregated_zone_config US US-Central
+"""
+import argparse
+from copy import deepcopy
+from pathlib import Path
+
+import yaml
+
+from electricitymap.contrib.config.model import Capacity, Zone
+
+NAME_MAPPING = {"hydro storage": "hydro_storage", "battery storage": "battery_storage"}
+
+
+def create_aggregated_config(zoneKey: str, timezone: str):
+    zone = Zone(
+        key=zoneKey,
+        sub_zone_names=[],
```

Comment:
Perhaps it would be better to stick with the `subZoneNames` naming scheme.

It also look like pylance (python type checking tool) expects it to be `subZoneNames`
![image](https://user-images.githubusercontent.com/30777521/200334501-f3f52a11-e908-4c59-94a9-1206b073cfc0.png)